javalib/src/main/scala/java/util/concurrent/CopyOnWriteArrayList.scala

```diff
@@ -150,7 +150,7 @@ class CopyOnWriteArrayList[E <: AnyRef] private (private var inner: js.Array[E])
   }
 
   def addAllAbsent(c: Collection[_ <: E]): Int = {
-    val elemsToAdd = c.iterator().filter(!contains(_)).toList
+    val elemsToAdd = c.iterator().filter(!contains(_)).toSet
```

Comment:
This is not correct. `addAllAbsent` guarantees that new elements will be added in the order returned by `c.iterator()`. The `toSet` would destroy that ordering.

What about a much simpler implementation altogether:
```scala
for (e <- c.iterator())
  addIfAbsent(e)
```
?